test/parallel/test-util-log.js

```diff
@@ -57,4 +62,4 @@ tests.forEach(function(test) {
 
 assert.strictEqual(process.stdout.writeTimes, tests.length);
 
-common.restoreStdout();
+restoreStdout();
```

Comment:
Unrelated but why isn't `restoreStderr()` also used?

Reply:
No idea.

Reply:
I remember signing off on this (https://github.com/nodejs/node/pull/13439)... I want to say it's related to the `mustNotCall` mechanism?...
But it should have been docu-commented for future reference.

Reply:
NJP (Not James's Problem)

Reply:
Sounds like a good first issue https://github.com/nodejs/node/issues/22472